src/test/java/Homework21.java

```diff
@@ -0,0 +1,65 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+
+public class Homework21 extends BaseTest {
+
+    @Test
+    public void renamePlaylist()  {
+        login("kbetestack@gmail.com", "te$t$tudent");
+        doubleClickChoosePlaylist();
+        enterPlaylistName();
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/214541296-772b4f0f-0523-4f12-978e-75ac24d3b81b.png)